command-expansion-server/src/worker.ts

```diff
@@ -110,3 +110,41 @@ export async function executeExpansion(opts: {
     return { activityInstance, commands: null, errors: [e?.message ?? 'Unexpected error'] };
   }
 }
+
+export async function executeEDSL(opts: { edslBody: string; commandTypes: string }): Promise<{
+  sequenceJson: SequenceSeqJson | null;
+  errors: ReturnType<UserCodeError['toJSON']>[];
+}> {
+  try {
+    const result = await codeRunner.executeUserCode(
+      opts.edslBody,
+      [],
+      'Sequence',
+      [],
+      3000,
```

Comment:
2 things:

1. I think 3 seconds for sequence authoring to execute is probably too lenient - if something is taking longer than maybe 1 second, I think it's more indicative of either a mistake in implementation of the sequence or problematic programming constructs. That said, 3 seconds isn't likely to be all that much more problematic than 1 second.
2. We should probably make these configuration variables at some point, not necessarily in this PR